docs/getting-started-guides/coreos/cloud-configs/master.yaml

```diff
@@ -26,7 +26,7 @@ coreos:
 
         [Service]
         ExecStartPre=-/usr/bin/mkdir -p /opt/bin
-        ExecStartPre=/usr/bin/wget -N -P /opt/bin https://storage.googleapis.com/k8s/setup-network-environment 
+        ExecStartPre=/usr/bin/wget -N -P /opt/bin https://github.com/kelseyhightower/setup-network-environment/releases/download/v1.0.0/setup-network-environment
```

Comment:
+1 Since we talked about giving control of the k8s bucket to the k8s team, we should just use the binary release from github.